doc/api.yml

```diff
@@ -563,16 +563,19 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  /students/fees:
     post:
       tags:
         - Paying
       summary: Create student fees
-      parameters:
-        - name: student_id
-          in: path
-          required: true
-          schema:
-            type: string
+      requestBody:
```

Comment:
This modification should be done in a separate commit as it has nothing to do with the fix we are trying to do. Next time, use ONE commit for doing ONE thing. This time, I will let it pass, BUT give an adequate name first.

`test: userUpser events` does NOT reflect the main purpose of this commit...

Reply:
For sure !
I have already created a new branch from origin dev, but I forgot to remove the api.yml ... my bad !